src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        String[] splitRowOfData = new String[4];
```

Comment:
it can be done like this
```suggestion
        String[] splitRowOfData;
```